Jet_fitting_3/examples/Jet_fitting_3/CMakeLists.txt

```diff
@@ -12,22 +12,18 @@ include(CGAL_Eigen3_support)
 if(TARGET CGAL::Eigen3_support)
   # Link with Boost.ProgramOptions (optional)
   find_package(Boost QUIET COMPONENTS program_options)
+
+  create_single_source_cgal_program("Mesh_estimation.cpp")
+  target_link_libraries(Mesh_estimation PUBLIC CGAL::Eigen3_support)
   if(Boost_PROGRAM_OPTIONS_FOUND)
-    if(TARGET Boost::program_options)
-      set(Boost_PROGRAM_OPTIONS_LIBRARY Boost::program_options)
-    endif()
-    if(CGAL_AUTO_LINK_ENABLED)
-      message(STATUS "Boost.ProgramOptions library: found")
+    target_compile_definitions(Mesh_estimation PRIVATE "CGAL_USE_BOOST_PROGRAM_OPTIONS")
```

Comment:
`CGAL_USE_BOOST_PROGRAM_OPTIONS` should be removed and the example not compiled if it is not available.